lib/shopify_transporter/shopify/product.rb

```diff
@@ -33,7 +33,8 @@ def header
             'Metafield Value Type', 'Variant Grams', 'Variant Inventory Tracker', 'Variant Inventory Qty',
```

Comment:
`'Variant Grams'` should be removed if we're replacing it with `'Variant Weight'`